java/core/src/java/org/apache/orc/impl/RecordReaderUtils.java

```diff
@@ -504,38 +504,33 @@ static void readDiskRanges(FSDataInputStream file,
     private static final class Key implements Comparable<Key> {
       private final int capacity;
       private final long insertionGeneration;
+      private final int hash;
 
-      Key(int capacity, long insertionGeneration) {
+      Key (int capacity, long insertionGeneration) {
```

Comment:
Nit: Is this extra space expected?

Reply:
Nope.  Not sure why I even did that.  Thanks for pointing it out.